sdk/identity/azure-identity/src/main/java/com/azure/identity/ClientCertificateCredentialBuilder.java

```diff
@@ -99,11 +97,12 @@ ClientCertificateCredentialBuilder enablePersistentCache() {
      * Specifies if the x5c claim (public key of the certificate) should be sent as part of the authentication request
      * and enable subject name / issuer based authentication. The default value is false.
      *
-     * @param includeX5c the flag to indicate if x5c should be sent as part of authentication request.
+     * @param sendCertificateChain the flag to indicate if certificate chain should be sent as part of authentication
+     * request.
      * @return An updated instance of this builder.
      */
-    public ClientCertificateCredentialBuilder includeX5c(boolean includeX5c) {
-        this.identityClientOptions.setIncludeX5c(includeX5c);
+    public ClientCertificateCredentialBuilder sendCertificateChain(boolean sendCertificateChain) {
+        this.identityClientOptions.setIncludeX5c(sendCertificateChain);
```

Comment:
For internal consistency you should consider changing the name on the identityClientOptions too